File: 3349-adjacent-increasing-subarrays-detection-i/3349-adjacent-increasing-subarrays-detection-i.java
class Solution {
    public boolean hasIncreasingSubarrays(List<Integer> nums, int k) {
        for(int i = 0; i <= nums.size() - 2*k; i++){
            boolean val = true;
            for(int j = i+1; j < i+k; j++){
                if(nums.get(j-1) >= nums.get(j) || nums.get(j+k-1) >= nums.get(j+k)){
                    val = false;
                    break;
                }
            }
            if(val) return true;
        }
        return false;
    }
}

File: 3349-adjacent-increasing-subarrays-detection-i/3349-adjacent-increasing-subarrays-detection-i.py
class Solution:
    def hasIncreasingSubarrays(self, nums: List[int], k: int) -> bool:
        for i in range(0,len(nums)-2*k+1):
            val=True
            for j in range(i+1,i+k):
                if(nums[j-1]>=nums[j]):
                    val=False
                    break
            if val:
                for l in range(i+k+1,i+2*k):
                    if(nums[l-1]>=nums[l]):
                        val=False
                        break
            if val:
                return True
        return False